src/main/java/net/bndy/wf/modules/core/api/UserController.java

```diff
@@ -129,6 +130,14 @@ public File upload(MultipartFile file, HttpServletRequest request) throws Illega
         return f;
     }
 
+    @RequestMapping(value = "/changePassword", method = RequestMethod.POST)
+    public void changePassword(@RequestBody Map<String, String> data) {
+        User u = this.getCurrentUser();
+        if (this.userService.comparePassword(data.get("oldPassword"), u.getPassword())) {
+            this.userService.changePassword(u.getId(), data.get("newPassword"));
+        }
```

Comment:
```java
else {
}
```
If password is error, should throw an exception.

Reply:
I will implement it later.